integreat_compass/cms/urls/protected.py

```diff
@@ -0,0 +1,7 @@
+"""
+URLconf for login-protected views of the cms package.
+"""
+
+# TODO: Add middleware to actually enforce access control on these views!
```

Comment:
Oh haha I wasn't aware that pylint fails with a TODO warning...
I guess we could also add the access control directly:
https://github.com/digitalfabrik/integreat-cms/blob/c0bbd6c40708aac4695d458906876286a6ced4d5/integreat_cms/core/middleware/access_control_middleware.py

Reply:
...oh, my solution was to just remove the word TODO, haha.

Reply:
But sure, we might as well.

Reply:
I have added a somehat trimmed down version of the one you linked. Hope it's OK like this. Also wondering if we should add a public app or find a different way to differentiate between which routes require auth and which do not

Reply:
I guess this depends on the general layout of the app - if the public pages are independent from the password protected pages, then yes it makes sense to put them in their own app, but if they are in a way part of the same functionality, then I'd argue that they should live in the same app...
I'm not really familiar with the thematic parts of the project, so do what you feel makes sense.